dashboard/job-pulse-kanban.html

```
@@ -0,0 +1,402 @@
+  .modal-close{float:right;background:none;border:none;color:var(--muted);cursor:pointer;font-size:1.1rem}
+</style>
+</head>
+<body>
+<header>
+  <h1>⚡ Job Pulse</h1>
+  <span id="status">—</span>
+  <div class="toolbar">
+    <button class="primary" onclick="fetchJobs()">⬇ Fetch Jobs</button>
+    <button onclick="runDryRun()">🔍 Dry-Run Submit</button>
+    <button onclick="exportState()">⬆ Export JSON</button>
+    <label style="cursor:pointer"><button as="span" onclick="document.getElementById('import-file').click()">⬇ Import JSON</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)